versions/versions.gradle

```diff
@@ -7,27 +7,27 @@
 ext.CONFIG.versions = [
     // Java compatibility target version
     java      : '1.7',
-    kotlin    : '1.2.0',
+    kotlin    : '1.3.21',
 
     // Android plugins, tools, SDK and libraries
     android   : [
         libraries: [
-            playServices: '11.0.4',
-            support     : '26.0.0',
+            playServices: '16.0.0',
```

Comment:
A couple of things regarding play services:
1. Play services should no longer have the same version for all of it's libraries ([see here](https://developers.google.com/android/guides/versioning)). And actually, some of the libraries only have a `v16.0.1` and no longer have `v16.0.0` available. 
2. Some SDKs that are using this value will break if updated past `11.0.4` for their play-services libraries.

So I think we should either leave it at `11.0.4` and then stop using it (just leave it up to each SDK to implement the version) or completely remove `playServices` from this repo and then possibly replace it with versions for the commonly used individual play-services libraries. What do you think?

Reply:
Nice catch, I will update it.

Reply:
@corycaywood Reverted back to 11.0.4

Reply:
it's still `16.0.0`